var/spack/repos/builtin/packages/cdo/package.py

```diff
@@ -42,6 +44,8 @@ class Cdo(AutotoolsPackage):
 
     variant('udunits2', default=True, description='Enable UDUNITS2 support')
     variant('libxml2', default=True, description='Enable libxml2 support')
+    # PROJ.6 supported starting 1.9.8.1
+    # https://launchpad.net/ubuntu/+source/cdo/+changelog
```

Comment:
I would move this comment down to the `depends_on` statements you modified below.